robsize.cc

```diff
@@ -405,6 +408,8 @@ void print_usage() {
     "\t--superfast\tRun at ludicrous speed which is even less accurate than --fast\n"
     "\t--write-asm\tPrint the raw generated instructions to a file and quit\n"
     "\t--list     \tList the available tests and their IDs\n"
+    "\t--start    \t(int) Value to start filler instruction count at.\n"
+    "\t--stop     \t(int) Value to stop filler instruction count at.\n"
```

Comment:
Usual usage syntax is like:

```
--start=START     Use START as the ...
```

Reply:
You should show the syntax for the argument like `--start=START` or `--start START` as UNIX convention. The other arguments aren't like that b/c they don't have arguments.

Reply:
Ah I see what you are saying. Is it fine if the rest of the args become misaligned, or should I still add spaces to the rest to keep them so?

Reply:
Add space.

Here's an example from `ls`:

```
Mandatory arguments to long options are mandatory for short options too.
  -a, --all                  do not ignore entries starting with .
  -A, --almost-all           do not list implied . and ..
      --author               with -l, print the author of each file
  -b, --escape               print C-style escapes for nongraphic characters
      --block-size=SIZE      scale sizes by SIZE before printing them; e.g.,
                               '--block-size=M' prints sizes in units of
                               1,048,576 bytes; see SIZE format below
  -B, --ignore-backups       do not list implied entries ending with ~
  -c                         with -lt: sort by, and show, ctime (time of last
                               modification of file status information);
                               with -l: show ctime and sort by name;
                               otherwise: sort by ctime, newest first
  -C                         list entries by columns
      --color[=WHEN]         colorize the output; WHEN can be 'always' (default
```

Reply:
Okay, should be fine now. Sorry I totally forgot that there was a convention to this after just being spoiled by python argparse handling all of it for me.

Reply:
@travisdowns - yeah maybe I should have mentioned this earlier, but I usually just use [this library](https://github.com/Taywee/args) to parse args in C++, because I hate the fiddly nature of doing it myself.

Reply:
Ooh that looks quite nice and user friendly! I really like boost:program_options too but a dep on boost for a project like this just feels wrong. I will keep Taywee/args in mind!